lms/djangoapps/learner_home/mock_data.json

```diff
@@ -79,45 +82,48 @@
                 "isPassing": true
             },
             "entitlement": null,
-            "courseRun": {
-                "isStarted": true,
-                "isArchived": false,
-                "startDate": "11/11/1900",
-                "endDate": "11/11/4040",
-                "minPassingGrade": 70,
-                "homeUrl": "edx.com/courses/my-course-url/home",
-                "marketingUrl": "edx.com/courses/my-course-url/marketing",
-                "progressUrl": "edx.com/courses/my-course-url/progress",
-                "unenrollUrl": "edx.com/courses/my-course-url/unenroll",
-                "resumeUrl": "edx.com/courses/my-course-url/resume",
-                "courseId": "course-number1-course-id1"
-            },
-            "certificate": {
-                "availableDate": null,
-                "isRestricted": false,
-                "isEarned": false,
-                "isDownloadable": false,
-                "certPreviewUrl": "edx.com/courses/my-course-url/cert-preview"
-            },
             "enrollment": {
-                "accessExpirationDate": "11/11/3030",
-                "canUpgrade": true,
-                "hasStarted": false,
                 "coursewareAccess": {
+                    "isTooEarly": true,
                     "hasUnmetPrerequisites": false,
-                    "isTooEarly": false,
                     "isStaff": false
                 },
+                "accessExpirationDate": "3030-11-11T00:00:00Z",
```

Comment:
Looks like there are 2 competing datetime precisions, is there a reason for this? `"accessExpirationDate": "3030-11-11T00:00:00Z"` vs `"lastEnrolled": "2022-09-20T14:35:27.049Z"`

Reply:
lastEnrolled is auto-generated to ensure variation in the courses.